docker-entrypoint.sh

```diff
@@ -4,7 +4,7 @@ set -e
 
 PLAIN='\033[0m'
 BOLD='\033[1;37m'
-
+CLONE_DIRECTORY="/tmp/gh-pages"
```

Comment:
This isn't a value that changes, so I'd rather not make it a variable. The reason why the other two are variables is because they look very strange otherwise.